packages/manager/apps/web/client/app/hosting/database/order-private/hosting-database-order-private.html

```diff
@@ -0,0 +1,158 @@
+<oui-back-button
+    data-on-click="$ctrl.goBack()">
+    <span data-translate="hosting_database_order_private_title"></span>
+</oui-back-button>
+
+<div data-ovh-alert="{{alerts.order}}"></div>
+
+<div class="text-center"
+        data-ng-if="$ctrl.loading.init">
+    <oui-spinner data-size="l"></oui-spinner>
+</div>
+
+<div data-ng-if="!$ctrl.loading.init">
+    <p data-translate="hosting_database_order_private_description"></p>
+
+    <oui-stepper name="sqlDatabaseOrderForm"
```

Comment:
```suggestion
    <oui-stepper data-name="sqlDatabaseOrderForm"
```
As the property is a custom one of the component

Reply:
No, it's the standard `name` attribute ;)